Clean up event loop from client (#58)
php-http/react-adapter: CHANGELOG.md

})();
```

> Another major change in this release is that injecting the event loop has been removed, and now fully uses

Reviewer: ```suggestion
Another major change in this release is that you no longer inject the event loop into the client. It now only uses
```

Author: Updated 👍
